src/analyses/mutexTypeAnalysis.ml

```diff
@@ -15,8 +15,9 @@ struct
   (* Removing indexes here avoids complicated lookups and allows to have the LVals as vars here, at the price that different types of mutexes in arrays are not dinstinguished *)
   module O = Offset.Unit
```

Comment:
Is this alias now used anywhere? Also, we should probably move the comment to where we now use `Mval.Unit`.

Reply:
It is in some conversions below, but I moved the comment around.